recipes/pysdl2/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "PySDL2" %}
+{% set version = "0.9.4" %}
+{% set md5 = "720b33ed048c169643406ae39ca4dd8d" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn:  {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  md5: {{ md5 }}
+  patches:
+    - locate_sdl.patch
+
+build:
+  number: 0
+  script: python setup.py install
+
+requirements:
+  build:
+    - patch  # [win]
+    - sdl2
+    - python
+    - setuptools
+
+  run:
+    - sdl2
+    - python
+
+test:
+  imports:
+    - sdl2
+
+about:
+  home: http://bitbucket.org/marcusva/py-sdl2
+  license: ZLIB
```

Comment:
So this is Public Domain where permissible. Would update this to `Public Domain/ZLIB`.